editors/zile/Portfile

```diff
@@ -1,26 +1,25 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem 1.0
+PortSystem          1.0
 
 name                zile
-version             2.4.14
-revision            1
+version             2.4.15
 categories          editors
 license             GPL-3+
 platforms           darwin
-maintainers         nomaintainer
+maintainers         {gmail.com:emanuele.giaquinta @exg} openmaintainer
 description         Zile Is Lossy Emacs
 long_description    \
```

Comment:
Minor nitpick - since the description's indentation has been fixed, you no longer need this slash / line-continuation here.

Basically `long_description` and "Zile is another Emacs-clone...." can be on the same line.